src/frontend/Sider/Journalføring/Standard/Dokumenter.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import DokumentPanel from './DokumentPanel';
+import { JournalføringState } from '../../../hooks/useJournalføringState';
+
+const Liste = styled.ul`
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+`;
+
+interface Props {
+    journalpostState: JournalføringState;
+}
+
+const Dokumenter: React.FC<Props> = ({ journalpostState }) => {
+    return (
+        <Liste>
+            {journalpostState.journalpost.dokumenter.map((dokument) => (
+                <li key={dokument.dokumentInfoId}>
+                    <DokumentPanel dokument={dokument} journalpostState={journalpostState} />
+                </li>
+            ))}
+        </Liste>
+    );
+};
```

Comment:
Trenger man ul og li her? Kan man fjerne de å bruke VStack direkte? 
```suggestion
const Dokumenter: React.FC<Props> = ({ journalpostState }) => {
    return (
        <VStack gap={'4'}>
            {journalpostState.journalpost.dokumenter.map((dokument) => (
                <DokumentPanel
                    key={dokument.dokumentInfoId}
                    dokument={dokument}
                    journalpostState={journalpostState}
                />
            ))}
        </VStack>
    );
};
```

Reply:
Definer "trenger". Hadde dette vært en løsning ut mot sluttbruker ville jeg definitivt svart ja, siden det blir semantisk riktig med tanke på UU. 

Men siden dette er en internløsning så er det kanskje ikke så farlig siden vi ikke trenger å tenke på UU.

Uavhengig så synes jeg det er en god praksis å bruke semantisk riktige html-tag så langt det lar seg gjøre, så jeg heller mot at det er best å beholde <ul> og <li>.

Reply:
Er det riktig når man har mer enn kun tekst, og komponenter inne i "listen" ? Burde ikke aria felter brukes då hvis det er pga UU?

Reply:
Men igjen, det er jo ikke sånn at vi bruker liste på langt nær i alle tilfeller der vi burde 🤔

Reply:
Det svarer ikke helt på spørsmålet. 
Uten å kunne dette bra, så føler jeg list er helt feil. Burde det i det minste være en `article`? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

Reply:
Dette kan jeg lite av. Når burde man egentligen bruke liste?
Burde det i være en article? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
Fortsatt usikker på om det burde være en liste. 

Dette svaret ga meg litt mer context til hvorfor lister generellt kan være fint https://stackoverflow.com/a/16299980
> The advantage for screen reader users is that screen readers can use the semantic markup to communicate useful information. For example, informing the user that they've just landed on a "list of 5 items" helps them to understand how the content is organized, and how much of it there is.

Samtidig skriver personen 
> That said, if each item in the list is very long or contains complex content with lots of nested elements, I agree @IanY that this could be confusing, as it then becomes difficult for users to keep track of where they are within the list.

Reply:
Ja, UU er jo et eget fag av en grunn :D Vil ikke kalle meg noen ekspert selv!